src/optimizer-page/scan-results/ScanResults.tsx

```diff
@@ -47,18 +47,18 @@ const ScanResults: FC<Props> = ({ data }) => {
       <div className="border-bottom border-light-400 mb-3">
         <header className="sub-header-content">
           <h2 className="sub-header-content-title">{intl.formatMessage(messages.scanHeader)}</h2>
-          <span className="locked-links-checkbox-wrapper">
-            <CheckBox
+          <Form.Group className="locked-links-checkbox-wrapper">
```

Comment:
It looks like `FormGroup` [acts as a `div` by default](https://github.com/openedx/paragon/blob/fd8a91c28342ff6aa7535f6541f49b3f50968653/src/Form/FormGroup.tsx#L25), where this was previously using a `span`.

I'd like to double check that everything appears/behaves correctly with this change.